components/tdisplays3/touchscreen/t_display_s3_touchscreen.cpp

```diff
@@ -162,5 +177,17 @@ void LilygoTDisplayS3Touchscreen::dump_config() {
   ESP_LOGCONFIG(TAG, "  Firmware version: %d", this->firmware_version_);
 }
 
+void LilygoTDisplayS3Touchscreen::sleep() { 
+  
+  this->interrupt_pin_->detach_interrupt();
+
+  this->reset_pin_->digital_write(false);
+  delay(5);
+  this->reset_pin_->digital_write(true);
+  delay(50);
```

Comment:
Hmm from the top of my head it was something like 150ms or so.. but in this case it's used to power-down the chip. You will only do this to minimize current draw during shutdown/sleep. 
This code therefore only runs when you are powering down as much as you can. The 50ms won't be a problem in this scenario, since your application is going to sleep, and not need to be fully responsive.